mainflux/errors.js

```diff
@@ -0,0 +1,161 @@
+        },
+        update: {
+            404: "Failed due to a non-existing user.",
+            401: "Missing or invalid access token provided.",
+            400: "Failed due to malformed JSON.",
+        },
+        update_user_tags:{
+            401: "Missing or invalid access token provided.",
+            400: "Failed due to malformed JSON.",
+        },
+        update_user_password:{
+            500: "Unexpected server-side error occurred.",
+            401: "Missing or invalid access token provided.",
+            400: "Failed due to malformed JSON.",
+        },
+        update_user_identity:{
+            401: "Missing or invalid access token provided.",
+            400: "Failed due to malformed JSON.",
+        },
+        update_user_owner:{
+            401: "Missing or invalid access token provided.",
+            400: "Failed due to malformed JSON.",
+            404: "A non-existent entity request.",
+        },
+        get_all: {
+            400: "Failed due to malformed query parameters.",
+            500: "Unexpected server-side error occurred.",
+            401: "Missing or invalid access token provided.",
+        },
+        disable: {
+            401: "Missing or invalid access token provided.",
+        },
+        enable: {
+            401: "Missing or invalid access token provided.",
+        },
+        memberships:{
+            401: "Missing or invalid access token provided.",
+            400: "Failed due to malformed query parameters.",
+        },
+        authorise_user: {
+            401: "Failed to perform authorisation.",
+            400: "Failed due to malformed JSON.",
+        },
+    }
+
+    groups= {
+        get : {
+            401: "Missing or invalid access token provided.",
+            404: "A non-existent entity request.",
+            400: "Failed due to malformed query parameters.",
+        },
+        create : {
+            401: "Missing or invalid access token provided.",
+            404: "A non-existent entity request.",
+            400: "Failed due to malformed query parameters.",
+        },
+        children : {
+            401: "Missing or invalid access token provided.",
+        },
+        parents : {
+            401: "Missing or invalid access token provided.",
+        },
+        update : {
+            401: "Missing or invalid access token provided.",
+        },
+        assign : {
+            500: "Unexpected server-side error occurred.",
+        },
+        unassign : {
+            401: "Missing or invalid access token provided.",
+            500: "Unexpected server-side error occurred.",
+        },
+        disable: {
+            401: "Missing or invalid access token provided.",
+        },
+        members:{
+            401: "Missing or invalid access token provided.",
+        },
+
+    }
+
+    things ={
+        create : {
+
+        },
+        get :{
+
+        },
+        get_all :{
+
+        },
+        
+    }
+
+    certs ={
```

Comment:
404 response message looks to repeated for all cases, this does not look correct